### ISSUE TITLE
please support screen reader

### ISSUE THREAD

**Issue by lih24338** [fcitx/fcitx5#876]
When I type using fcits, the screen reader cannot read the word candidate panel, does not interpret phrases, and gives no feedback, and when I use shortcut keys to switch input methods, the screen reader does not report which input method I am currently in.
 - Tried input methods: fcits-rime, fcits-Chinese-addons
 - Operating system:archlinux
 - Desktop environment: gnome45
 - Display service:wayland
 - Log: No need to provide

**Comment by lih24338** [comment]
[orca source code](https://github.com/GNOME/orca)

**Comment by wengxt** [comment]
@lih24338 need to support at-spi not orca
